tools/DynamoPackagesAnalyzer/Readme.txt

```diff
@@ -0,0 +1,60 @@
+This tool requires analyzebinaries experimental feature of upgrade-assistant enabled
+https://github.com/dotnet/upgrade-assistant#experimental-features
+
+This tool has three operation modes:
+
+1. zipfile
+This mode expects and arrays of packages zip archives, to be analyzed one by one
+
+Flags explanation:
+--files: used to receive the files to process
+-l: whether to lookup for package details at dynamopackages.com
```

Comment:
what does this flag do - I'm not getting it from this readme.

Reply:
When this flag is present, the package list is downloaded, and using the information from pkg.json on each of the zip packages, specifically the name property, the process tries to get the complete information about the package from the downloaded list

Reply:
Yes, I'm updating the readme